test.R

```diff
@@ -1,7 +1,8 @@
-library(DSI)
 library(DSMolgenis)
 library(dsBaseClient)
 
+dsGetInfo(molgenis())
```

Comment:
How should this work? I get an error: `Error in molgenis() : could not find function "molgenis"`

Reply:
This does work:
`dsGetInfo(new("MolgenisDriver"))`